### DISCUSSION TITLE
What's Clementines project state?

### DISCUSSION THREAD

**Discussion by Clusters** [clementine-player/Clementine#7231] [Q&A] [answered]
The last official release of Clementine was 6 years ago, but the project itself seems to be quite alive. I couldn't find any info about it so I have to ask here:
What's the state of this project and which version (or release) of Clementine should be used?

**Comment by fitojb** [comment, marked as answer]
You should use the daily builds; it’s what Debian packages for obvious reasons.

**Reply by Anglodel** [reply]
Interesting.
However, what prevents Clementine from having real official new releases ? Is there an internal roadmap that have to be ended for a new release or something ?
